react-client/src/components/GameOutcome.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class GameOutcome extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+  }
+
+  render() {
+
+    identities = []
+    // Here, will have to itterate over players, creating <Identity> compenents.
```

Comment:
if the identities array were given by the server, it could be rendered as a list in this component. Don't think we need another component